src/staking-v3/logic/models/DappStaking.ts

```diff
@@ -172,13 +172,7 @@ export interface TiersConfiguration {
   readonly numberOfSlots: number;
```

Comment:
Perhaps the `numberOfSlots` field can be removed here too.

Reply:
I prefer to leave it because it is used on the portal UI. The variable is now calculated as below
`numberOfSlots: configuration.slotsPerTier.reduce((acc, val) => acc + val.toNumber()`